src/parser.y

```diff
@@ -49,6 +49,7 @@ struct lexer_param;
 %token INVALID_CHARACTER
 %token <literal> IDENT
 %token <literal> FIELD
+%token <literal> VARIABLE
```

Comment:
This could be called `SYMBOL` instead of `VARIABLE` maybe

Reply:
How about `BINDING`?